.husky/pre-commit

```diff
@@ -1,5 +0,0 @@
-#!/bin/sh
-. "$(dirname "$0")/_/husky.sh"
-
-npx lint-staged
-yarn test
```

Comment:
IMO this is the only line that should be removed, the rest is actually quite helpful and unobtrusive.